dali/sasha/saxref.cpp

```diff
@@ -2748,6 +2810,7 @@ class CSashaXRefServer: public ISashaServer, public Thread
         }
         IArrayOf<IGroup> groupsdone;
         StringArray dirsdone;
+        bool saveToDebugPlane = isContainerized() && props->getPropBool("@saveToDebugPlane", true);
```

Comment:
I think it would make more sense for this to be a member (of CNewXRefManagerBase), instead of parsing into saveToEclWatch, addBranch, addErrorsWarnings.